Documentation/migration-scripts/python-scripts/metabase-group-script/fetching_and_auto_mapping_groups.py

```
@@ -0,0 +1,337 @@
+            "password": config.get_string("metabase.password")
+        }
+
+    except Exception as e:
+        print(
+            f"Error parsing configuration at '{unified_conf}': {e}",
+            file=sys.stderr
+        )
+        sys.exit(1)
+
+def get_session(url, username, password):
+    res = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)